extensions/2.0/Khronos/KHR_materials_diffuse_transmission/README.md

```diff
@@ -0,0 +1,423 @@
+</em><br>
+<p style="font-size:0.7em;">(candle model by <a href="https://sketchfab.com/lucatorcigliani">lucatorcigliani</a> with modifications by @emackey. Original <a href="https://sketchfab.com/3d-models/candle-in-a-glass-ec6cf358c4dd4cbda7be67f7846cbc9e">source</a> CC-BY 4.0)<p>
+</figcaption>
+</figure>
+</div>
+
+
+## Extending Materials
+
+The effect is activated by adding the `KHR_materials_diffuse_transmission` extension to any glTF material.
+
+```json
+{
+    "materials": [
+        {
+            "extensions": {
+                "KHR_materials_diffuse_transmission": {
+                    "diffuseTransmissionFactor": 0.25,
+                    "diffuseTransmissionTexture": {
+                      "index": 0
+                    },
+                    "diffuseTransmissionColorFactor": [
+                      1.0,
+                      0.9,
+                      0.85
+                    ]
+                }
+            }
+        }
+    ]
+}
+```
+
+## Properties
+
+|                                     | Type                                                                | Description                                                                                                                                                                     | Required                 |
+|-------------------------------------|---------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|
+| **diffuseTransmissionFactor**       | `number`                                                            | The percentage of non-specularly reflected light that is diffusely transmitted through the surface.                      | No, default: `0`         |
+| **diffuseTransmissionTexture**      | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | A texture that defines the percentage of non-specularly reflected light that is diffusely transmitted through the surface. Stored in the alpha (`A`) channel. Will be multiplied by the diffuseTransmissionFactor.                 | No                       |
+| **diffuseTransmissionColorFactor**  | `number[3]`                                                         | The color that modulates the transmitted light.                                                                                                                                 | No, default: `[1, 1, 1]` |
+| **diffuseTransmissionColorTexture** | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | A texture that defines the color that modulates the diffusely transmitted light, stored in the `RGB` channels and encoded in sRGB. This texture will be multiplied by diffuseTransmissionColorFactor. | No                       |
+
```

Comment:
Since these properties follow the regular `*Factor`/`*Texture` glTF pattern, the four subsequent sections should be refactored as two: "Diffuse Transmission" and "Diffuse Transmission Color" without using property names as headings.

_Similar change is needed for `KHR_materials_transmission` but that's irrelevant here._